packages/react-form-renderer/src/files/field.d.ts

```diff
@@ -9,6 +9,8 @@ export interface FieldActions {
   [key: string]: FieldAction;
 }
 
+export type ResolvePropsFunction = (props: AnyObject, metaInput: AnyObject, formOptions: AnyObject) => AnyObject;
```

Comment:
meta and form options types should be defined. You can use the final form meta props definition.

Reply:
Oh I missed formOptions, I will add it tomorrow.